### DISCUSSION TITLE
Question about default settings

### DISCUSSION THREAD

**Discussion by gremlinsGizmo** [Pryaxis/TShock#1838] [Q&A]
I just started using Tshock today, I only want to use the server to turn off corruption spread because it bugs me, but are all settings in the config. Jason default settings? like if you were to load up a normal world and a server would it be the same because that's all I really want from it

**Comment by hakusaro** [comment]
Your best bet is the [Vanillafier plugin](https://github.com/Pryaxis/Vanillafier).
